src/client-types.ts

```diff
@@ -148,3 +148,23 @@ export type PromptTemplateExecuteInput = z.input<
 export type PromptTemplateExecuteOutput =
   ApiTypes.PromptTemplateExecuteOutput['results'];
 export type PromptTemplateExecuteOptions = HttpHandlerOptions;
+
+// HISTORY
+
+export const HistoryInputSchema = ApiTypes.HistoryInputSchema.pick({
+  status: true,
+  origin: true,
+})
+  .extend({
+    count: z.number().int().min(1).nullish(),
+    offset: z.number().int().min(0).nullish(),
+  })
+  .partial();
+export type HistoryInput = z.input<typeof HistoryInputSchema>;
+
+export type HistoryOptions = HttpHandlerOptions;
+
+export const HistoryOutputSchema = ApiTypes.HistoryOutputSchema;
+export type HistoryOutput = z.output<
+  typeof HistoryOutputSchema
+>['results'][number];
```

Comment:
Not like this, the type MUST correspond to the schema. Use `.shape.results.element`